bigbluebutton-html5/imports/api/bbb/index.js

```diff
@@ -44,4 +44,6 @@ class htmlBBB {
 
 }
 
-window.BBB = new htmlBBB();
+if (window.BBB == undefined) {

```

Comment:
What do you think of adding this snippet inside a `init()` function?

```
export const init = () => {
  if (window.BBB == undefined) {
    window.BBB = new BBB();
  }
};

```
Feels weird to import something to add it to global scope.